From 49d7a3e44b54f18b0e2762d8dbb087ad4c2f0276 Mon Sep 17 00:00:00 2001
From: Yann Dirson <ydirson@altern.org>
Date: Thu, 15 Jun 2006 09:48:08 +0200
Subject: [PATCH] Call cvs with -q flag when fetching the log

This gets rid of the neverending output polution cvs sends to stderr,
which is mostly useless and has a strong potential to hide real problem
reports
---
 cvsps.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cvsps.c b/cvsps.c
index d94dfe0..be700ba 100644
--- a/cvsps.c
+++ b/cvsps.c
@@ -301,12 +301,12 @@ static void load_from_cvs()
 	 * which is necessary to fill in the pre_rev stuff for a 
 	 * PatchSetMember
 	 */
-	snprintf(cmd, BUFSIZ, "cvs %s %s %s -d '%s<;%s' %s", compress_arg, norc, ltype, date_str, date_str, use_rep_buff);
+	snprintf(cmd, BUFSIZ, "cvs %s %s -q %s -d '%s<;%s' %s", compress_arg, norc, ltype, date_str, date_str, use_rep_buff);
     }
     else
     {
 	date_str[0] = 0;
-	snprintf(cmd, BUFSIZ, "cvs %s %s %s %s", compress_arg, norc, ltype, use_rep_buff);
+	snprintf(cmd, BUFSIZ, "cvs %s %s -q %s %s", compress_arg, norc, ltype, use_rep_buff);
     }
     
     debug(DEBUG_STATUS, "******* USING CMD %s", cmd);
-- 
1.6.5.GIT

